homeassistant/components/sensor/hp_ilo.py

```diff
@@ -115,20 +135,25 @@ def device_state_attributes(self):
 
     def update(self):
         """Get the latest data from HP ILO and updates the states."""
+        import jsonpath_rw
```

Comment:
Please don't include a new dependency for extracting data. We use Jinja2 templates for this.